lib/dialects/mssql/query-generator.js

```diff
@@ -570,7 +570,14 @@ module.exports = (function() {
 
       if (options.limit || options.offset) {
         if (!options.order || (options.include && !subQueryOrder.length)) {
-          fragment += ' ORDER BY ' + this.quoteIdentifier(model.primaryKeyAttribute);
+          if(!isSubQuery && options.order) {
+            fragment += ', ';
```

Comment:
The issue here is that the query for the added test ends in `ORDER BY title DESC ORDER BY [id] OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY;`, which is invalid MSSQL syntax. 

The correct query should end in `ORDER BY title DESC, [id] OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY`, which this PR solves.

I agree with rewriting the code. :)